ajax/libs/vega/package.json

```diff
@@ -13,8 +13,13 @@
     {
       "basePath": "",
       "files": [
-        "vega.js",
-        "vega.min.js"
+        "vega*.+(js|js.map)"
+      ]
```

Comment:
Yes. However, we may make more v2 patch releases in the future. I just wanted to make sure that cdnjs cover v2 and v3 releases.